packages/fx-core/templates/plugins/resource/aad/bicep/parameters.json

```diff
@@ -1,8 +1,8 @@
 {
-  "aadClientId": {
+  "m365ClientId": {
     "value": "{{FX_RESOURCE_AAD_APP_FOR_TEAMS__CLIENTID}}"
   },
-  "aadClientSecret": {
+  "m365ClientSecret": {
```

Comment:
need to update related code in function plugin as well

Reply:
Fixed.